src/types/AST.js

```diff
@@ -13,6 +13,11 @@ type NodePath<T> = {
  * AST types
  */
 
+interface LongNumber {
```

Comment:
Could we rename it to `I64Literal` or `i64`?

What do you think of https://github.com/xtuc/js-webassembly-interpreter/pull/70#discussion_r160047149? hi and low as I32 seems a bit overkill tho

Reply:
> Could we rename it to `I64Literal` or `i64`?

I don't think `I64Literal` makes sense. The current `NumberLiteral` interface describes an AST node type, whereas `LongNumber` describes the representation of a 'long' number as having `high` and `low` properties.

The corresponding literal, that describes an AST node that contains a long value is [further down the same file](https://github.com/xtuc/js-webassembly-interpreter/pull/79/files#diff-f0fcb9b6c38a452cae948d4b45ab85ffR50).

Perhaps `i64` would make sense, although I feel it would look a little out of place with the naming for the other interfaces / types in this file.

> What do you think of #70 (comment)? hi and low as I32 seems a bit overkill tho

This does feel overkill to me to. Unless I'm mistaken, the information these interfaces convey would actually be lost in the AST itself? i.e. it would yield exactly the same output.

Reply:
> I don't think I64Literal makes sense

Yes you are right, I'm :+1: for `LongNumberLiteral` AST and `LongNumber` for the runtime instance. 

Having I64 = [i32, i32] is not very useful from the AST point of view, we just need i64 = [number, number]

Reply:
done :-) https://github.com/xtuc/js-webassembly-interpreter/pull/79/commits/128ac7973bd3bc0262adc9d9ae62435f73567f1a